application/src/test/java/uk/gov/hmcts/ccd/definition/store/flyway/FlywayNoOpStrategyTest.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.ccd.definition.store.flyway;
+
+import org.flywaydb.core.Flyway;
+import org.flywaydb.core.api.MigrationInfo;
+import org.flywaydb.core.api.MigrationInfoService;
+import org.flywaydb.core.api.MigrationState;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.autoconfigure.flyway.FlywayMigrationStrategy;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.reset;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FlywayNoOpStrategyTest {
+
+    @Mock
+    private Flyway flyway;
+
+    @Mock
+    private MigrationInfoService infoService;
+
+    @Mock
+    private MigrationInfo info;
+
+    private final FlywayMigrationStrategy strategy = new FlywayNoOpStrategy();
+
+    @After
+    public void tearUp() {
+        reset(flyway, infoService, info);
+    }
+
+    @Test
+    public void should_not_throw_exception_when_all_migrations_are_applied() {
```

Comment:
Is there a reason for the use of snake_case over camelCase for method names in this file?

Reply:
haha.. it's a copy paste file. Let me fix it